
# HG changeset patch
# User tiwai
# Date 1187185081 -7200
# Node ID b230fef1ea76192e8cfaba252beaaca369ab085b
# Parent 114e579b6ce1ac8c38f2b7ea43f34501db1b626e
alsactl - Remove debug print

Remove an old debug print that simply annoys.

--- a/alsactl/state.c	Tue Aug 07 15:36:41 2007 +0200
+++ b/alsactl/state.c	Wed Aug 15 15:38:01 2007 +0200
@@ -810,10 +810,8 @@ static int add_user_control(snd_ctl_t *h
 		err = snd_ctl_elem_add_integer(handle, id, count, imin, imax, istep);
 		if (err < 0)
 			goto error;
-		if (tlv) {
-			fprintf(stderr, "XXX write TLV...\n");
+		if (tlv)
 			snd_ctl_elem_tlv_write(handle, id, tlv);
-		}
 		break;
 	case SND_CTL_ELEM_TYPE_BOOLEAN:
 		err = snd_ctl_elem_add_boolean(handle, id, count);

